R/sp_get_dbms_data_dictionary.R

```diff
@@ -0,0 +1,17 @@
+#' Download a table from the dbms and create a data dictionary data frame
+#'
+#' @param table_name The name of a table to be processed.
+#' @param con A DBI connection object for obtaining the local copy to be processed
+#'
+#' @return a data frame
+#' @export
+#' @importFrom dplyr %>%
+#' @importFrom dplyr tbl
+#' @importFrom dplyr collect
+#'
+#' @examples all_meta <- sp_get_dbms_data_dictionary("rental")
+sp_get_dbms_data_dictionary <- function(table_name) {
+  table_ptr <- tbl(con, table_name)
```

Comment:
I do not see `con` being passed as a parameter, only `table_name`. Where is `con` defined? If it is defined as a global somewhere, I think it would be better to pass it as an argument.

Reply:
I added `con` back in.  Removed it at some point, but had left it in the `@param` list.